src/internal/m365/graph/middleware.go

```diff
@@ -148,7 +148,8 @@ func (mw *LoggingMiddleware) Intercept(
 		logBody = logger.DebugAPIFV ||
 			os.Getenv(logGraphRequestsEnvKey) != "" ||
 			resp.StatusCode == http.StatusBadRequest ||
-			resp.StatusCode == http.StatusForbidden
+			resp.StatusCode == http.StatusForbidden ||
+			resp.StatusCode == http.StatusConflict
```

Comment:
Might wanna update the comment to reflect this.

Reply:
doesn't seem like a very helpful comment.  I've removed the 4xx enumeration part of it.